lib/docker.ts

```diff
@@ -158,8 +208,8 @@ export const startNextcloud = async function(branch = 'master', mountApp: boolea
 export const configureNextcloud = async function(apps = ['viewer'], vendoredBranch?: string) {
 	vendoredBranch = vendoredBranch || _serverBranch
 
-	console.log('\nConfiguring nextcloud...')
-	const container = docker.getContainer(CONTAINER_NAME)
+	console.log('\nConfiguring nextcloud…')
```

Comment:
```suggestion
	console.log('\nConfiguring Nextcloud…')
```
Should be capitalized?